Userland/Applications/Browser/Tab.cpp

```diff
@@ -45,7 +45,9 @@ String g_search_engine = {};
 URL url_from_user_input(const String& input)
 {
     if (input.starts_with("?") && !g_search_engine.is_null()) {
-        return URL(String::formatted(g_search_engine, urlencode(input.substring(1))));
+        String query = input.substring(1);
+        query.replace(" ", "+", true);
+        return URL(String::formatted(g_search_engine, urlencode(query)));
```

Comment:
This seems odd - so when I search for `1 + 1`, it'll replace that with `1   1`? Please at least explain the reasoning for this change in the commit message :)